common/src/main/java/org/dash/wallet/common/data/PresentableTxMetadata.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.common.data
+
+import android.graphics.Bitmap
+import androidx.room.Ignore
+import org.bitcoinj.core.Sha256Hash
+
+data class PresentableTxMetadata(
+    var txId: Sha256Hash,
+    var memo: String = "",
+    var service: String? = null,
+    var customIconId: Sha256Hash? = null
+) {
+    @Ignore var icon: Bitmap? = null
+}
```

Comment:
A convenience class to only get the metadata we want to display on the Home screen.